package.json

```diff
@@ -21,15 +21,15 @@
     "test:node": "node node_test.js"
   },
   "devDependencies": {
-    "browserify": "^11.0.0",
+    "browserify": "^13.1.0",
     "chrome-launch": "^1.1.3",
     "self-signed-https": "^1.0.5",
-    "send": "^0.12.2",
+    "send": "^0.14.1",
```

Comment:
I didn't see anything that looked breaking in the changelog. (And it's a dev dependency, so if the tests pass, this update should be ok.)